### PR TITLE
Patch to enhance netperf script

### PR DESCRIPTION
**Pull request by hathyaga**
Enhancement contains a variable to allow user to
choose the download location for netperf tool

Signed-off-by: Harsha Thyagaraja <harshkid@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by hathyaga** [comment]
Job log
[Netperf_joblog_with_user_specified_netperf_download_parameter_PASS.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/1085274/Netperf_joblog_with_user_specified_netperf_download_parameter_PASS.txt)

File: io/net/netperf_test.py.data/README.txt [hunk 1]
```diff
@@ -12,6 +12,12 @@ Iface		        - interface on which test run
 timeout		        - Timeout
 NETSERVER_RUN	    - Whether to run netserver in peer or not
 EXPECTED_THROUGHPUT - Expected Throughput as a percentage (1-100)
+netperf_download    - User has the option to choose download location
+                      for netperf tool.
+                      User can use ftp download location passed as default
+                      to this variable or use IO-FVT repo
+                      "http://9.47.67.137/repo/netperf-2.7.0.tar.bz2"
```
Comment (narasimhan-v) [inline io/net/netperf_test.py.data/README.txt]: Please remove this example location

Comment (hathyaga) [inline io/net/netperf_test.py.data/README.txt]: Will do. Thanks

**Comment by hathyaga** [comment]
Change suggested is implemented and amended to the same commit. Please merge

**Pull request merged**